manager/manager.py

```diff
@@ -249,31 +249,23 @@ def start_job(self, job_id):
                     if error.startswith('Bad Request: '):
                         error = error[len('Bad Request: '):]
                     job.error_message(error)
-
-            if 'Payload' in json_data:
-                payload = json_data['Payload']
-
-                print('Payload:')
-                print(json.dumps(payload))
-
-                for message in payload['log']:
-                    if message:
-                        job.log_message(message)
-                for message in payload['errors']:
-                    if message:
-                        job.error_message(message)
-                for message in payload['warnings']:
-                    if message:
-                        job.warning_message(message)
-
-                success = payload['success']
-
-                if payload['errors']:
-                    job.log_message('{0} function returned with errors.'.format(module.name))
-                elif payload['warnings']:
-                    job.log_message('{0} function returned with warnings.'.format(module.name))
-                elif payload['log']:
-                    job.log_message('{0} function returned.'.format(module.name))
+                elif 'success' in json_data:
```

Comment:
Yup, there is another response possible than an exception in our Lambda function being thrown or the conversion not being a success, and that would be if the server couldn't be connected (timeout), etc. I look to see if "success" is in the payload, as that tells me we actually have a valid response from the converter before I call json_data["success"]. If "success" doesn't exist, then it will throw an exception.